common/geth/failover.go

```diff
@@ -0,0 +1,60 @@
+package geth
+
+import (
+	"sync"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+)
+
+type FailoverController struct {
+	NumberFault     uint64
+	NumberSuccess   uint64
```

Comment:
How is this used?
It looks like we don't really `ProcessError` if `err == nil` in `MultiHomingClient`. Is this necessary?
If this is not being used, can we remove it?

Reply:
NumberFault is used for deciding which rpc to use by "MultihomeCleint.GetRPCInstance"

Reply:
NumberSuccess is not used